fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -0,0 +1,72 @@
+module FastlaneCore
+  class ToolCollector
+    TOOLS = %w(fastlane fastlane_core deliver snapshot frameit pem sigh produce cert gym pilot credentials_manager spaceship scan supply watchbuild match screengrab)
+    HOST_URL = "https://fastlane-enhancer.herokuapp.com/"
+
+    attr_reader :error
+
+    def did_launch_action(name)
```

Comment:
`action` is actually a tool here, right? Depending on if we want to use this in `fastlane` too, we might want to rename this, or at add a comment describing the parameter.